wcfsetup/install/files/lib/system/background/BackgroundQueueHandler.class.php

```diff
@@ -20,6 +20,12 @@
  */
 final class BackgroundQueueHandler extends SingletonFactory
 {
+    /**
+     * Indicates that the client should trigger a check for
+     * pending jobs in the background queue.
```

Comment:
This explanation should rather be moved into the getter’s PHPDoc, because the latter is part of the public API.